digdag-standards/src/main/java/io/digdag/standards/command/kubernetes/KubernetesClientConfig.java

```diff
@@ -46,24 +51,49 @@ private static KubernetesClientConfig createFromSystemConfig(final Optional<Stri
         }
         final String keyPrefix = KUBERNETES_CLIENT_PARAMS_PREFIX + clusterName + ".";
         final Config extracted = validateParams(StorageManager.extractKeyPrefix(systemConfig, keyPrefix));
-        return create(clusterName,
-                extracted.get("master", String.class),
-                extracted.get("certs_ca_data", String.class),
-                extracted.get("oauth_token", String.class),
-                extracted.get("namespace", String.class));
+        if (extracted.get("use_kube_config", boolean.class, false)){
+            io.fabric8.kubernetes.client.Config kubeConfig;
+            kubeConfig = getKubeConfigFromPath(extracted.get("kube_config_path", String.class));
+            return create(clusterName,
+                kubeConfig.getMasterUrl(),
+                kubeConfig.getCaCertData(),
+                kubeConfig.getOauthToken(),
+                kubeConfig.getNamespace()
+              );
+        }else{
+            return create(clusterName,
+                    extracted.get("master", String.class),
+                    extracted.get("certs_ca_data", String.class),
+                    extracted.get("oauth_token", String.class),
+                    extracted.get("namespace", String.class));
+        }
     }
 
     private static Config validateParams(final Config config)
     {
-        if (!config.has("master") ||
-                !config.has("certs_ca_data") ||
-                !config.has("oauth_token") ||
-                !config.has("namespace")) {
-            throw new ConfigException("kubernetes config must have master:, certs_ca_data:, oauth_token: and namespace:");
+        if (!(config.has("master") &&
+                config.has("certs_ca_data") &&
+                config.has("oauth_token") &&
+                config.has("namespace")) &&
+                !(config.has("use_kube_config") &&
+                 config.has("kube_config_path"))
+            ) {
+            throw new ConfigException("kubernetes config must have master:, certs_ca_data:, oauth_token: and namespace: Or use_kube_config: kube_config_path");
```

Comment:
nitpick: `"kubernetes config must have master:, certs_ca_data:, oauth_token: and namespace:, or use_kube_config: 
 and kube_config_path:"`